troubleshooting/troubleshooting-ios-cannot-load-iframe.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Cannot Load iframe on iOS Devices When Using Cordova 5.0.0 or Later
+description: Cannot load iframe on iOS devices with Cordova 5.0.0 and 5.2.0.
+type: troubleshooting
+page_title: Cordova 5.0.0 or Later Apps do not Load iframe on iOS
+slug: iframe-not-loading-ios
+position: 
+tags: appbuilder, ios, cordova 
+teampulseid: 
+ticketid: 1060248
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+  <tr>
+    <td>Cordova framework version</td>
+    <td>Cordova 5.0.0 and above</td>	
+  </tr>
+</table>
+
+## Description
+
+Apps that are deployed on an iOS device do not show the content of the iframe.
+
+## Steps to Reproduce
+
+1. [Create](http://docs.telerik.com/platform/appbuilder/cordova/creating-your-app/creating-a-new-app) or open a Cordova application.
+1. [Change the target Cordova version](http://docs.telerik.com/platform/appbuilder/cordova/configuring-your-app/configure-target-framework) to 5.0.0 or later.
+1. Include an `<iframe src=""></iframe>` tag to an HTML view.
+1. Deploy the app to an iOS device.
+
+## Cause
+
+The new Cordova domain whitelisting system provides a configurable security policy. It lets you define which sites will be accessible by your app. For more information, see the [Cordova Whitelist Guide](https://cordova.apache.org/docs/en/latest/guide/appdev/whitelist/).
```

Comment:
Cordova 5.0.0(Cordova for iOS 4.0.1) incorporates the Cordova whitelisting system, which provides a configurable security policy. As Cordova 5.0.0 is marked experimental in AppBuilder, the project template does not include the required configuration to enable whitelisting.